### PR TITLE
[MINI-3848] Demo app analytics

### PR DESCRIPTION
**Pull request by rleojoseph**
# Description
* Updated code to send sample app analytics.

## Links
MINI-3848

# Checklist
- [x] I have read the [contributing guidelines](CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

**Review by Climbatize — APPROVED** [review]
LGTM

**Review by munir-rakuten — APPROVED** [review]
Looks good to me, I have commented one suggestion though.👍

File: Example/Analytics/DemoAppAnalytics.swift [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    case appear = "appear"
+    case click = "click"
+    case pageLoad = "pv"
+}
+
+internal enum DemoAppRATActionType: String, CaseIterable {
+    case open
+    case close
+    case changeStatus
+    case initial
```
[the remark this answers is not in the document]
Comment (rleojoseph) [reply Example/Analytics/DemoAppAnalytics.swift:14]: @munir-rakuten , default is a reserved keyword. Thought of not using it unless if it is unavoidable.

[the remark this answers is not in the document]
Comment (Climbatize) [reply Example/Analytics/DemoAppAnalytics.swift:14]: What about using a `String` enum where initial = "default" @rleojoseph ?

**Review by corycaywood — APPROVED** [review]

**Pull request merged**